src/main/java/httpServer/http/request/RequestBuilder.java

```diff
@@ -0,0 +1,81 @@
+package httpServer.http.request;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static httpServer.http.Constants.crlf;
+
+public class RequestBuilder {
+
+    public String method;
+    public String resource;
+    public String httpVersionNumber;
+    public Map<String, String> headers = new HashMap<>();
+    public String body;
+
+    public RequestBuilder buildRequestLine(String line) {
+        String[] requestLine = line.split(" ");
+        this.method = requestLine[0];
+        this.resource = requestLine[1];
+        this.httpVersionNumber = requestLine[2].replace(crlf, "");;
+        return this;
+    }
+
+    public RequestBuilder buildHeaderLine(String line) {
+        String[] requestLine = line.split(": ");
+        String removedCLDR = requestLine[1].replace(crlf, "");
```

Comment:
Probably more readable to just do `String[] requestLine = line.trim().split(": ");`